### DISCUSSION TITLE
Strange characters in verbose output.

### DISCUSSION THREAD

**Discussion by buhtz** [pytest-dev/pytest#10623] [Q&A] [answered]
![grafik](https://user-images.githubusercontent.com/11861496/210093546-15d428f9-d94d-45cb-81b4-d9b2080a7e8d.png)

Do you see the three inverted question marks behind `test_image_file_to_bytes`?

I checked my whole code and can't find any debug-like `print()` statement. What could this be? Any idea?

**Comment by antonblr** [comment, marked as answer]
@buhtz - these � (REPLACEMENT CHARACTER used to replace an unknown, unrecognized, or unrepresentable character) are coming from the underlying https://github.com/python-pillow/Pillow library. 

Somewhere down this line
https://codeberg.org/buhtz/buhtzology/src/commit/691c07c7272cd5889e943bfccc0797b1110ac5f0/tests/test_report.py#L26
```python
img.save(pathlib.Path('img.bmp'))
```

Changing the file name to e.g. `img.png` eliminates the problem. Perhaps ask in python-pillow/Pillow.
